solr/core/src/java/org/apache/solr/servlet/CoordinatorHttpSolrCall.java

```diff
@@ -215,6 +216,9 @@ public HttpSolrCall createInstance(
         boolean retry) {
       if ((path.startsWith("/____v2/") || path.equals("/____v2"))) {
         return new CoordinatorV2HttpSolrCall(this, filter, cores, request, response, retry);
+      } else if (path.startsWith("/.sys.COORDINATOR-COLL")) {
```

Comment:
After fixing the endless loop, a new issue arose in the test where certain calls that are part of adding a replica, like `/admin/ping` were failing since they were attempted to be proxied but were missing a collection. This just handles calls for the synthetic collection normally.